test/login.restore.spec.js

```diff
@@ -0,0 +1,62 @@
+// spec.js
+describe('Login and restore password. ', function () {
+
+  browser.ignoreSynchronization = true;
+  // browser.driver.manage().window().setSize(1920, 1080);
+  browser.driver.manage().timeouts().implicitlyWait(20000);
+  browser.waitForAngularEnabled(false);
+  it('Should proceed to the https://www.indiegogo.com page', () => {
+    browser.get('https://www.indiegogo.com');
+    expect(browser.getTitle()).toEqual('Crowdfund Innovations & Support Entrepreneurs | Indiegogo');
+
+  });
+
+  it('Should test that cookie consent checkbox are present and unticked by default', () => {
+    expect(element(by.id('CybotCookiebotDialogBodyContentCheckboxPersonalInformation')).isSelected().false);
+    element(by.css('.CybotCookiebotDialogBodyContentLabelPersonalInformation')).click();
+    expect(element(by.id('CybotCookiebotDialogBodyContentCheckboxPersonalInformation')).isSelected().true);
```

Comment:
It will be a good idea to make a separate variable for `CybotCookiebotDialogBodyContentCheckboxPersonalInformation` string.